### PR TITLE
Add plots

### PR DESCRIPTION
**Pull request by awirb**
* add option to include title of colorbar, textbox with list of ava names 
* add plot option for scatter plots of max values 
* add example runScript 
* add option to specify release area scenario in ini file
* add third release area scenario for HockeySmoothChannel
* add info in docu
* adjust logging

### CODE REVIEW

File: avaframe/out3Plot/plotUtils.py [hunk 1]
```diff
@@ -209,24 +209,33 @@ def saveAndOrPlot(cfgPath, cfgFlags, outFileName, fig):
...
 def putAvaNameOnPlot(ax, avaDir):
     '''
-    Puts the date and avalanche name in the lower left corner of the given
+    Puts the date and avalanche name (or a list of ava names) in the lower left corner of the given
     matplotlib axes
     '''
-    avaName = pathlib.PurePath(avaDir).name
 
-    infoText = datetime.datetime.now().strftime("%d.%m.%y") + \
+    if isinstance(avaDir, str):
```
Comment (fso42) [inline avaframe/out3Plot/plotUtils.py:229]: Please add a short comment what this if statement switches

File: avaframe/out3Plot/statsPlots.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
+"""
+    plot statistics of simulations
+
+"""
+
+# load python modules
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+import glob
```
Comment (fso42) [inline avaframe/out3Plot/statsPlots.py:10]: Not used? -> Remove

File: avaframe/out3Plot/statsPlots.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+from avaframe.in3Utils import fileHandlerUtils as fU
```
Comment (fso42) [inline avaframe/out3Plot/statsPlots.py:11]: Not used? -> Remove

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/out3Plot/statsPlots.py (hunk 2 above)
Comment (awirb) [reply avaframe/out3Plot/statsPlots.py:10]: done

File: avaframe/out3Plot/statsPlots.py (hunk 3 above)
Comment (awirb) [reply avaframe/out3Plot/statsPlots.py:11]: done

File: avaframe/out3Plot/plotUtils.py (hunk 1 above)
Comment (awirb) [reply avaframe/out3Plot/plotUtils.py:229]: Added a comment

File: avaframe/com1DFA/com1DFACfg.ini [hunk 4]
```diff
@@ -15,6 +15,8 @@ flagEnt = True
 # True if resistance areas shall be included
 flagRes = True
 flagDev = False
+# specify a particular release area scenario, provide name of shapefile
```
Comment (fso42) [inline avaframe/com1DFA/com1DFACfg.ini:18]:
```suggestion
# specify a particular release area scenario, provide name of shapefile (optional)
```

File: avaframe/in3Utils/logUtils.py [hunk 5]
```diff
@@ -11,7 +11,7 @@
 
 def writeCfg2Log(cfg, cfgName='Unspecified'):
     """ write a configparser object to log file"""
-    
+
     log = logging.getLogger('root')
```
Comment (fso42) [inline avaframe/in3Utils/logUtils.py:15]:
```suggestion
    log = logging.getLogger('avaframe')
```

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/com1DFA/com1DFACfg.ini (hunk 4 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFACfg.ini:18]: done

File: avaframe/in3Utils/logUtils.py (hunk 5 above)
Comment (awirb) [reply avaframe/in3Utils/logUtils.py:15]: done

**Review by fso42 — APPROVED** [review]

**Pull request merged**